conda/recipes/nvstrings/meta.yaml

```diff
@@ -26,17 +26,19 @@ build:
 requirements:
   build:
     - cmake
+    - python 3.6.* # [py36]
+    - python 3.7.* # [py37]
   host:
-    - python
+    - python x.x
     - setuptools
     - numba>=0.40
   run:
-    - python
+    - python x.x
     - setuptools
 
 test:
   requires:
-    - python
+    - python x.x
```

Comment:
I think we can drop `x.x` here.

Reply:
Can I just drop the Python entirely since that's a runtime requirement anyway?

Reply:
Yep. Good point :)